sinfonier-backend-api/apiRequests/RequestHandler.py

```diff
@@ -3,13 +3,17 @@
 
 class Error(dict):
     def __init__(self, msg):
-        super(Error).__init__()
+        #super(Error).__init__()
```

Comment:
Yes, you right, I haven't note that the self param are missed,  this should work `super(Error,self).__init__()`